CHANGELOG.md

```diff
@@ -6,6 +6,34 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [2.4.0] - Not released
+### Added
+- Users can now write posts to public and protected groups that they are not members of.
+- Group administrators can now block certain users from write to the group. This
+  feature includes:
+  - New API methods for group administrators:
```

Comment:
Do we want this available in api tokens?

Reply:
Sure, https://github.com/FreeFeed/freefeed-server/pull/577/files#diff-4c8123b27e1814c4c39b43a03173957b390a7af60ab49c2d4906eb9e89910930